ang/civicase/ActivityPanel.js

```diff
@@ -60,6 +62,35 @@
         element.find('.crm-submit-buttons a.edit').addClass('btn btn-primary');
       }
 
+      /**
+       * Set scrollbar for activity panel body
+       */
+      function setPanelBodyScrollbar () {
```

Comment:
what does it mean to "set the scrollbar"? One would imagine you're talking about some kind of scrollbar vendor plugin, but that's not the case by looking at the implementation. I think the title should be changed to better reflect what's going on here

Reply:
Renamed to `setPanelBodyHeight`.